feature-flags.json

```diff
@@ -3,5 +3,6 @@
 	"serviceSetupV2",
 	"swgModule",
 	"userInput",
-	"unifiedDashboard"
+	"unifiedDashboard",
+	"dashboardSharing"
```

Comment:
A minor nitpick here, but the rest of these are alphabetised, so it'd be nice to put this one at the top of the file 😅